ocs_ci/deployment/deployment.py

```diff
@@ -1709,17 +1662,64 @@ def deploy(self):
         run_cmd_multicluster(
             f"oc create -f {constants.OPENSHIFT_DR_SYSTEM_OPERATORGROUP}"
         )
+        self.deploy_dr_hub_operator()
 
         # RBD specific dr deployment
         if self.rbd:
             rbddops = RBDDRDeployOps()
             rbddops.deploy()
             self.meta_obj.deploy_and_configure()
 
-        self.deploy_dr_hub_operator()
-        self.deploy_dr_cluster_operator()
+        self.deploy_dr_policy()
         self.meta_obj.conf.update({"dr_policy_name": self.dr_policy_name})
-        self.meta_obj.update_s3_profile()
+        self.update_ramen_config_misc()
+
+    def update_ramen_config_misc(self):
+        config_map_data = self.meta_obj.get_ramen_resource()
+        self.update_config_map_commit(config_map_data)
+
+    def update_config_map_commit(self, config_map_data, prefix=None):
+        """
+        merge the config and update the resource
+
+        Args:
+            config_map_data (dict): base dictionary which will be later converted to yaml content
+            prefix(str): Used to identify temp yaml
+
+        """
+        # Ramen section's value will be a string rather than type dict
+        # So we have to load that string , convert back to dict and update
+        # config_map_data
```

Comment:
Do we need to explain these steps in comments? If so I think these could be converted to more concise debug statements. I.E `logger.debug("Converting ramen data to dict")`

Same for the large comments below.

Reply:
done